packages/fs.go

```diff
@@ -73,10 +95,12 @@ func NewZipPackageFileSystem(p *Package) (*ZipPackageFileSystem, error) {
 	var root string
 	found := false
 	for _, f := range reader.File {
-		name := filepath.Clean(f.Name)
-		parts := strings.Split(name, string(filepath.Separator))
-		if len(parts) == 2 && parts[1] == "manifest.yml" {
-			root = parts[0]
+		name := path.Clean(f.Name)
+		dir, base := path.Split(name)
+		dir = path.Clean(dir)
+		// Find manifest files at the root of the package
+		if dir != "." && path.Dir(dir) == "." && base == "manifest.yml" {
+			root = dir
 			found = true
```

Comment:
Nit. Not related to this change, but `found` is redundant here, right? We can also check if `root != ""`.

Reply:
True, I had assumed that `found` was used for extra readability.